integration-tests/__tests__/__snapshots__/coverage_remapping.test.js.snap

```diff
@@ -24,149 +25,153 @@ Object {
     },
     "branchMap": Object {
       "0": Object {
+        "line": 4,
```

Comment:
I don't have any idea if these are good or bad changes...

Reply:
Probably bad, as they don't match the original #2290

Reply:
These shouldn't change from 75% to 83.3%. This means the remapping is broken.

Reply:
Any changes here are probably bad. I think the HTML report is written for this, we just don't capture it, so this test could be run on master and this branch and the HTML report fidelity checked as far as coverage lining up.

We could use a headless browser to screenshot the HTML report and do an image comparison instead of this indecipherable snapshot, but not sure how stable it would be

Reply:
We might want to keep `this._sourceMapStore.transformCoverage` in, but only use it if it successfully remaps something.

Reply:
I think it returning empty since it couldn't find the mappings for that file in the first place, in `map`